python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py

```diff
@@ -500,7 +494,22 @@ def process_gtfs_rt_files(db_manager: DatabaseManager) -> None:
                     paths=files["tu_paths"],
                     db_manager=db_manager,
                 )
+
+            if vp_events.shape[0] > 0 and tu_events.shape[0] > 0:
+                # combine events when available from tu and vp
                 events = combine_events(vp_events, tu_events)
+            elif tu_events.shape[0] == 0:
+                # no tu events found
+                events = vp_events
+                events["tu_stop_timestamp"] = None
+            elif vp_events.shape[0] == 0:
+                # no vp events found
+                events = tu_events
+                events["vp_move_timestamp"] = None
+                events["vp_stop_timestamp"] = None
+            else:
+                # no events found
+                events = pandas.DataFrame()
```

Comment:
I saw a few edge cases in our seed file processing that I think makes this work flow make more sense. We were assuming that if we have a parquet file for a specific hour we're processing, we would get events records from that file, but I don't think that is always the case.